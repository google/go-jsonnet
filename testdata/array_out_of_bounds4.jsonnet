[1,2,3][42]