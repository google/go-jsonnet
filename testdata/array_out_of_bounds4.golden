RUNTIME ERROR: Index 42 out of bounds, not within [0, 3)
-------------------------------------------------
	testdata/array_out_of_bounds4:1:1-12	$

[1,2,3][42]

-------------------------------------------------
	During evaluation	


